Utils.py

```diff
@@ -8,11 +8,11 @@
 from contextlib import contextmanager, closing
 
 
-COALA_KEY = "coala-sublime"
+coala_key = "coala-sublime"
```

Comment:
nooooo ... CONSTANTS should always be capitalised.  That is PEP 8.

I suggest giving this constant a different name.  `APP_NAME` ?